pigeon/notifications.dart

```diff
@@ -36,19 +36,30 @@ class NotificationChannel {
   final Int64List? vibrationPattern;
 }
 
+/// Corresponds to `android.content.Intent`
+///
+/// See:
+///   https://developer.android.com/reference/android/content/Intent
+///   https://developer.android.com/reference/android/content/Intent#Intent(java.lang.String,%20android.net.Uri,%20android.content.Context,%20java.lang.Class%3C?%3E)
+class AndroidIntent {
+  AndroidIntent({required this.action, required this.dataUrl, required this.flags});
```

Comment:
nit:
```suggestion
  AndroidIntent({required this.action, required this.dataUrl, this.flags = 0});
```

This isn't a parameter of the underlying constructor, so effectively it's optional in the underlying API. (One has to set it later after construction.) So making it optional here seems the most faithful to that API.